frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/cluster-setup-getting-started-card.tsx

```diff
@@ -26,9 +27,9 @@ export const ClusterSetupGettingStartedCard: React.FC = () => {
     return null;
   }
 
-  const moreLinkURL = window.SERVER_FLAGS.documentationBaseURL
-    ? `${window.SERVER_FLAGS.documentationBaseURL}html/post-installation_configuration/post-install-machine-configuration-tasks`
-    : 'https://docs.okd.io/latest/post_installation_configuration/machine-configuration-tasks.html';
+  const moreLinkURL = getDocumentationURL(
```

Comment:
was this broken by the linter? If not lets make it into a one=liner

Reply:
Linter insists on multi-line.